docs/en/operations/server-configuration-parameters/settings.md

```diff
@@ -2019,12 +2307,104 @@ Keys:
 
 **Recommended usage**
 
-``` xml
+```xml
 <send_crash_reports>
     <enabled>true</enabled>
 </send_crash_reports>
 ```
 
+## ssh_server
+
+Configuration for writing the public part of a host key to a known_hosts file on the SSH client side on the first connect.
+
+Host Key Configurations are inactive by default. 
+Uncomment the host key configurations, and provide the path to the respective ssh key to active them:
+
+Example:
+
+```xml
+<ssh_server>
+    <host_rsa_key>path_to_the_ssh_key</host_rsa_key>
+    <host_ecdsa_key>path_to_the_ssh_key</host_ecdsa_key>
+    <host_ed25519_key>path_to_the_ssh_key</host_ed25519_key>
+</ssh_server>
+```
+
+## storage_configuration
+
+Allows for multi-disk configuration of storage. It can be specified in `config.xml` or in a separate file in directory `config.d`.
```

Comment:
Let's remove the second sentence. It applies to _all_ settings and clear anyways.